vendor.conf

```diff
@@ -0,0 +1,50 @@
+# grpc and protobuf
```

Comment:
I see.

It's probably okay to leave it as is, if it is not automatically generated. If we want to tweak the format to include sources, we'd need to have some sort of directive in a comment.

Imagine if we wanted to import from a fork:

```
github.com/pkg/errors v1.0 abcdef git@github.com/stevvooe/errors
```

There would need to be a way to detect these syntax changes. Another options is to simply support keyed data on the end:

```
github.com/pkg/errors v1.0 abcdef source=git@github.com/stevvooe/errors
```

I can drop this from the requirements if we just want to go forward.